src/scripts/main.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const wallRect = wall.getBoundingClientRect();
```

Comment:
Ensure that the '.wall' element exists in the DOM. If it doesn't, 'wall' will be null, and calling 'getBoundingClientRect()' will throw an error.